logs_sdk/lib/opentelemetry/sdk/logs/log_record_data.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # LogRecordData is a Struct containing {LogRecord} data for export.
+      LogRecordData = Struct.new(:timestamp,             # optional Integer nanoseconds since Epoch
+                                 :observed_timestamp,    # Integer nanoseconds since Epoch
+                                 :severity_text,         # optional String
+                                 :severity_number,       # optional Integer
+                                 :body,                  # optional String, Numeric, Boolean, Array<String, Numeric,
+                                 #   Boolean>, Hash{String => String, Numeric, Boolean,
+                                 #   Array<String, Numeric, Boolean>}
+                                 :attributes, # optional Hash{String => String, Numeric, Boolean, Array<String, Numeric, Boolean>}
+                                 :trace_id,              # optional String (16-byte binary)
+                                 :span_id,               # optional String (8-byte binary)
+                                 :trace_flags,           # optional Integer (8-bit byte of bit flags)
+                                 :resource,              # optional OpenTelemetry::SDK::Resources::Resource
+                                 :instrumentation_scope, # OpenTelemetry::SDK::InstrumentationScope
+                                 :total_recorded_attributes) do # Integer
+        def unix_nano_timestamp
```

Comment:
nit: is that possible to do `alias_method :unix_nano_observed_timestamp, :unix_nano_timestamp`?

Reply:
Good question! The current setup [follows the pattern for export helpers in the SpanData class](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/sdk/lib/opentelemetry/sdk/trace/span_data.rb#L34-L50).

They're not exactly aliasable because the `unix_nano_timestamp` method references the `timestamp` attribute on the LogRecordData Struct, whereas the other method uses the `observed_timestamp` attribute. I could refactor it to accept an argument, but then we'd be breaking from the pattern in SpanData. 

I'd be curious to hear what approach the maintainers prefer we take.

Reply:
Whoops, commented on this before I saw Francis' comment. These methods will be removed.